src/less/core/global.less

```diff
@@ -21,15 +21,15 @@
 // Body reset
 
 html {
-    // font-size: 10px;
+    font-size: 16px;
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
     min-height: 100%;
 }
 
 body {
     font-family: @font-stack;
     font-size: @base-font-size;
-    line-height: 1.75rem;
+    line-height: 24px;
```

Comment:
Could we use on that would relative to the elements font-size `1.5`?
This will makes the line 24px for paragraph as spec-ed, but will also apply a sane default for other elements. If we don't do it this way, we will almost always have to set the line-weight when we are modifying an elements font-size.
https://stackoverflow.com/questions/12303169/line-height-without-units/12304722#12304722